CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Changed
 
 - Updated review-database to 0.31.0.
+- Updated `nodeStatusList` GraphQL API response to include `nameDraft`,
+ `profile`, `profileDraft`, and `gigantoDraft`, offering a more comprehensive
+ view of the node’s status.
```

Comment:
This alignment doesn't look good. I suspect you didn't install Prettier.

Reply:
Thank you for the check, and I found that we might need to add this:
<img width="498" alt="image" src="https://github.com/user-attachments/assets/68f3761f-2c32-4671-ac29-487b636db598">

I had been using Prettier for about a month, but it seems like it did not work as I expected. However, with the addition of this setting, I was able to get the desired result.

Reply:
I guess you turned off Prettier manually by clicking the icon at the right bottom corner. You can turn it on by clicking it again or setting it in the file as you did.